client/state/reader-ui/selectors.js

```diff
@@ -32,3 +34,13 @@ export function getPersistedLastActionPriorToLogin( state ) {
 	}
 	return state.readerUi?.persistedLastActionPriorToLogin;
 }
+
+/**
+ * Temporary selector to check if the reader multi-site version of the reader dashboard is enabled
+ * @returns {boolean} Whether the user is enabled for the reader multi-site dashboard
+ */
+export function isReaderMSDEnabled() {
```

Comment:
Should we move this to `client/reader/utils.ts`. This file seems particular to Redux.